src/dashboard/form-manager/components/FieldRenderer.tsx

```
@@ -0,0 +1,64 @@
+import { getFieldConfigFromFactory } from '../factories';
+import { FormField } from '../types';
+import { checkDependency } from '../utils';
+
+export const getFieldConfig = ( field: FormField ) => {
+    const mappedField = {
+        ...field,
+        label: (
+            <div className="flex gap-1 items-center">
+                <span
+                    className="dokan-form-field-label"
+                    dangerouslySetInnerHTML={ {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)